tools/docker/Dockerfile

```diff
@@ -1,6 +1,23 @@
 FROM alpine
 ARG TARGETARCH
-RUN apk add --no-cache --verbose ca-certificates && adduser -D kopia && addgroup kopia kopia
+
+# install CA certificates and create user/group 'kopia' with IDs 51515:51515
+RUN apk add --no-cache --verbose ca-certificates && \
+  addgroup -g 51515 kopia && \
+  adduser -u 51515 -H -D -G kopia kopia && \
+  mkdir -p /logs /cache /config && \
+  chown kopia:kopia /logs /cache /config
```

Comment:
Unless there's good reason I'd not create a bunch of directories in the root. 

The script here can be simplified (ideally no RUN statements). That enables miltiarch building

Reply:
this is already multiarch for 3 linux archs, Windows won't work, though.